steamclog/src/main/java/com/steamclock/steamclog/Destinations.kt

```diff
@@ -36,7 +38,7 @@ internal class CrashlyticsDestination : Timber.Tree() {
         companion object {
             fun with(message: String): NonFatalException {
                 val stackTrace = Thread.currentThread().stackTrace
-                val numToRemove = 10 // Move down the stack past Timber and Steamclog calls.
+                val numToRemove = 12 // Move down the stack past Timber and Steamclog calls.
```

Comment:
Can we file a ticket for adding unit tests for verifying this is working as expected? If we forget to update this number and push it to a client app, it could mess up our logging pretty badly.

Reply:
https://github.com/steamclock/steamclog-android/issues/57

Reply:
NonFatalException was a work around to support Crashyltics quirks, so once we have fully removed support for it we can remove this as well.